app/assets/javascripts/backbone/views/breakeven_product_total_view.coffee

```diff
@@ -0,0 +1,104 @@
+class MS.Views.BreakevenProductTotalView extends Backbone.View
+
+  # The view is called from the editiable table view.
+
+  # events:
+  #   'click tr [data-action="delete"]': 'removeRow'
+  #   'click .actions [data-action="add"]': 'addRow'
+
+  initialize: (options) ->
+    @products = options.products
+
+    @totalFixedCosts = parseFloat(@$('.total_fixed_costs').val())
+
+    _.each @products, (product) =>
+      product.on "product:changed", () =>
+        @calculateTotals()
+
+    # Defer loading of totals until the dom is ready
+    $ =>
+      @calculateTotals()
```

Comment:
This whole view should not be loaded until the DOM is ready, right? Why is this needed?